app/BUILD.bazel

```diff
@@ -348,6 +353,40 @@ android_library(
     visibility = ["//visibility:private"],
 )
 
+# Library for layout resource files
+android_library(
+    name = "databinding_resources",
+    custom_package = "org.oppia.app.databinding",
+    manifest = "src/main/DbAndroidManifest.xml",
+    srcs = BINDING_ADAPTER_FILES,
```

Comment:
Please file an issue if you haven't already to migrate our adapter files back to Kotlin. This will need to be post-phase 2 since it will require a lot of changes in Bazel to support a proper Kotlin android_library rule.

Reply:
Okay!